### PR TITLE
upgrade xiami to V2

### PR DESCRIPTION
**Pull request by fakelbst**
Just upgrade xiami to V2 :)

### CODE REVIEW

File: core/connectors.js [hunk 1]
```diff
@@ -282,12 +282,12 @@ define(function() {
 			js: ['connectors/radioplusbe.js']
 		},
 
-		{
-			label: 'Songza',
-			matches: ['*://songza.com/*'],
-			js: ['connectors/v2/songza.js'],
-			version: 2
-		},
+		// {
```
Comment (inverse) [inline core/connectors.js]: Why is this here?

**Comment by fakelbst** [comment]
@inverse  I'm sorry I marked songza as commented by mistake. Just fixed it. Thanks.

**Comment by fakelbst** [comment]
Also upgrade 163 music to V2 :)
